### ISSUE TITLE
When enabling shadows also show datepicker and timeline

### ISSUE THREAD

**Issue by meh9** [TerriaJS/terriajs#4199]
At the moment, when we enable Shadows for 3D datasets it uses the current date and time by default. This means the results will always be different, and you actually want to be able to change the date and/or time so you can see how the shadows change - this is the core of the feature.

Whenever you enable Shadows, we should also put in the datepicker so that you can control the exact time, and also enable the timeline at the bottom so you can drag around and animate the shadows.

Also see related: https://github.com/TerriaJS/nsw-digital-twin/issues/317

**Comment by na9da** [comment]
Similar request from community - https://github.com/TerriaJS/terriajs/discussions/7245

**Comment by mattpgreen** [comment]
yes - it  is a fundamental aspect of shadow cast/receive analysis that the user can check for date/time scenarios across the year as we travel around the and to our eyes it  does its north south north travels.
can this please get  into  the backlog?
